### DISCUSSION TITLE
Cron Job

### DISCUSSION THREAD

**Discussion by ArthurLee9** [dvershinin/fds#35] [General]
I checked in:

 /etc/cron.daily for a file named FDS and it wasn't there.

What is the best way to reinstall the FDS app on RHEL 8, please?

It is possible I may have deleted it because I didnt understand what its job was.

**Comment by dvershinin** [comment]
`dnf reinstall fds` should do the job.

**Comment by ArthurLee9** [comment]
***@***.*** ~]# dnf reinstall fds

Updating Subscription Management repositories.

Last metadata expiration check: 1:06:15 ago on Mon 08 May 2023 11:48:28 AM EDT.

Installed package fds-0.0.30-1.el8.noarch (from getpagespeed-extras-noarch) not available.

Error: No packages marked for reinstall.

 

 

***@***.*** ~]# dnf install fds

Updating Subscription Management repositories.

Last metadata expiration check: 1:06:47 ago on Mon 08 May 2023 11:48:28 AM EDT.

Package fds-0.0.30-1.el8.noarch is already installed.

Dependencies resolved.

Nothing to do.

Complete!

 

 

From: Danila Vershinin ***@***.***>
Reply-To: dvershinin/fds ***@***.***>
Date: Monday, May 8, 2023 at 12:48 PM
To: dvershinin/fds ***@***.***>
Cc: ArthurLee9 ***@***.***>, Author ***@***.***>
Subject: Re: [dvershinin/fds] Cron Job (Discussion #35)

 

dnf reinstall fds should do the job.

**Comment by dvershinin** [comment]
`Installed package fds-0.0.30-1.el8.noarch (from getpagespeed-extras-noarch) not available.` means you have disabled the repository.
